pipeline_dsl/test/test_pipeline.py

```diff
@@ -26,7 +26,7 @@ def concourse_ctx():
 @patch.object(sys, "argv", ["test"])
 class TestPipeline(unittest.TestCase):
     def test_init_task(self):
-        with Pipeline("test", script_dirs={"fake": "fake_scripts"}) as pipeline:
+        with Pipeline("test", script_dirs={"fake": "fake_scripts", "fake_glob": "fake_scripts/*.yaml"}) as pipeline:
```

Comment:
`script_dirs=["fake_scripts/*.yaml"]` won't work, will it?

Reply:
Same like in the first answer.